### PR TITLE
Update futuniuniu from 10.12.724_202011101415 to 10.13.738,202012151016

### PR DESCRIPTION
**Pull request by chunheisiu**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Welcome, and thank you, but that’s not the version they show on the website.

**Comment by chunheisiu** [comment]
The website I sourced from is https://www.futunn.com/download/mac and it seems like they are a version ahead of what is used by brew currently.

**Review by vitorgalvao — DISMISSED** [review]

**Comment by miccal** [comment]
> The website I sourced from is https://www.futunn.com/download/mac and it seems like they are a version ahead of what is used by brew currently.

This is for an `app` called `Futubull.app` -- is this a replacement of the original `FutuNiuniu.app`?

**Comment by chunheisiu** [comment]
> > The website I sourced from is https://www.futunn.com/download/mac and it seems like they are a version ahead of what is used by brew currently.
> 
> This is for an `app` called `Futubull.app` -- is this a replacement of the original `FutuNiuniu.app`?

They are most likely the same app but localized for a different market.

**Comment by vitorgalvao** [comment]
> This is for an `app` called `Futubull.app` -- is this a replacement of the original `FutuNiuniu.app`?

From the original website you can get to the new one; so they seem connected and that’s why I made the further changes.

File: Casks/futuniuniu.rb [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
   name "FutuNiuniu"
   name "富途牛牛"
   desc "Trading application"
-  homepage "https://hk.futu5.com/"
+  homepage "https://www.futunn.com/"
 
   # Renamed for consistency: app name is different in the Finder and in a shell.
-  # Original discussion: https://github.com/Homebrew/homebrew-cask/pull/7435
-  app "FutuNiuniu.app", target: "富途牛牛.app"
+  app "FutuNiuniu.app", target: "Futubull.app"
```
Comment (vitorgalvao) [inline Casks/futuniuniu.rb:13]: We’ll need to rename the cask.

Comment (miccal) [reply Casks/futuniuniu.rb]: Done.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**